tests/unit/store.spec.ts

```diff
@@ -165,7 +174,7 @@ describe('getter tests', () => {
         { name: 'rename', oldname: 'foo', newname: 'bar' },
         { name: 'rename', oldname: 'baz', newname: 'spam' },
       ];
-      const state = buildState({ pipeline });
+      const state = buildState({ ...stateWithOnePipeline(pipeline) });
```

Comment:
```suggestion
      const state = buildState(stateWithOnePipeline(pipeline));
```

Reply:
Thanks for noticing these which have passed my bulk replace vigilance :see_no_evil: